src/BqField.cpp

```diff
@@ -253,6 +270,30 @@ class BqField {
     case BQ_RECORD:
       SET_VECTOR_ELT(x, i, recordValue(v));
       break;
+    case BQ_GEOGRAPHY:
+      if (v.IsString()) {
+        Rcpp::RObject chr = Rf_mkCharLenCE(v.GetString(), v.GetStringLength(), CE_UTF8);
```

Comment:
I'm not sure if this matters that much, but I think CHARSXP maps to a Rcpp String object?

```suggestion
        Rcpp::String chr = Rf_mkCharLenCE(v.GetString(), v.GetStringLength(), CE_UTF8);
```

Reply:
In that case, I think its fine as is. `RObject` is a catch-all and you aren't doing anything fancy with it, so it should be fine!